From 8cf68ae028aa7898ff85519a300d8466bd557ead Mon Sep 17 00:00:00 2001
From: Simon van der Veldt <simon.vanderveldt@gmail.com>
Date: Wed, 11 Sep 2019 19:44:10 +0200
Subject: [PATCH] slack.desktop: Get rid of absolute path for Icon

---
 usr/share/applications/slack.desktop | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/usr/share/applications/slack.desktop b/usr/share/applications/slack.desktop
index ec8d121..3567251 100644
--- a/usr/share/applications/slack.desktop
+++ b/usr/share/applications/slack.desktop
@@ -4,7 +4,7 @@ StartupWMClass=Slack
 Comment=Slack Desktop
 GenericName=Slack Client for Linux
 Exec=/usr/bin/slack %U
-Icon=/usr/share/pixmaps/slack.png
+Icon=slack
 Type=Application
 StartupNotify=true
 Categories=GNOME;GTK;Network;InstantMessaging;
-- 
2.21.0

